### PR TITLE
Changing the wording on board bios. Fix #18.

### LINKED ISSUE #18: Update Member Information
- Change wordings of member blurbs
- Add fun facts :)

### PR DESCRIPTION
**Pull request by chenss3**

### CODE REVIEW

File: src/components/about/members.json [hunk 1]
```diff
@@ -3,46 +3,46 @@
...
+        "enjoy": "It's so cool to see a community of hackers getting together for a weekend to achieve one common goal of producing an interesting product."
     },
     {
         "name": "Aadarsh Jha",
         "image": "/images/headshots/Aadarsh-Jha.jpg",
         "role": "President, Mentor",
         "from": "Austin, TX",
         "year": "Junior",
-        "enjoy": "Coming together and making something cool. "
+        "enjoy": "It's such an honor meeting such talented individuals - specifically, my co-president Sayd. I cherish these days to be able to work along side such a breath-taking individual."
```
Comment (saydus) [inline src/components/about/members.json:24]: I approve

Comment (saydus) [reply src/components/about/members.json:24]: This is amazingly worded. Best PR

**Review by saydus — APPROVED** [review]
Looks good!

File: src/components/about/members.json (hunk 1 above)
Comment (aadarshjha) [reply src/components/about/members.json:24]: how does one un approve 👎 😢 😭

**Comment by SamLee514** [comment]
Thanks so much Sophia! Fantastic change.

**Comment by aadibajpai** [comment]
perfectionist PR ❤️

**Pull request merged**